xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -741,6 +757,63 @@ def main():
                 block, vpr_loc, "Constraining clock block {}".format(block)
             )
 
+    """ Constrain IDELAYCTRL sites
+
+    Prior to the invocation of this script, the IDELAYCTRL sites must have been
+    replicated accordingly to the IDELAY specifications.
+    There can be three different usage combinations of IDELAYCTRL and IDELAYs in a design:
+        1. IODELAYs and IDELAYCTRLs can be constrained to banks as needed.
+           Manual replication of the constrained IDELAYCTRLs is necessary to provide a controller for each bank.
+        2. IODELAYs and a single IDELAYCTRL can be left entirely unconstrained, becoming a default group.
+           The IDELAYCTRLis replicated depending on bank usage. Replication must have happened prior to this step
+        3. One or more IODELAY_GROUPs can be defined that contain IODELAYs and a single IDELAYCTRL each.
```

Comment:
We should clarify in the comment that #3 is not supported at this time.

Reply:
Done